module_report/check_modules.py

```diff
@@ -0,0 +1,53 @@
+import importlib
+from pathlib import Path
+import platform
+import subprocess
+
+
+lib_path = Path([*Path('.venv').rglob('site-packages')][0])
+libs = ['PyQt5', 'PyQt6', 'PySide2', 'PySide6']
+qtpy_files = [f.name[:-len('.py')] for f in Path('qtpy').glob('*.py') if f.name.startswith('Qt')]
+
+
+modules = {}
+
+
+for lib in libs:
+    path = lib_path / lib
+    files = path.glob('*.pyi')
+    for f in files:
+        if f.name.startswith('Qt'):
+            name = f.name[:-len('.pyi')]
+            if name not in modules:
+                modules[name] = {'libs':[], 'missing': name not in qtpy_files}
+            modules[name]['libs'].append(lib)
+
+
+module_names = sorted(modules.keys())
+
+
+false_positives = [
+    'QtChart',
+    'QtOpenGLFunctions',
+    'QtScript',
+    'QtScriptTools',
+]
+
+
+for name in false_positives:
+    if name in module_names:
+        module_names.remove(name)
+    if name in modules:
+        modules.pop(name)
+
+
```

Comment:
```suggestion
```

Move this above to do the check inline; I'm not sure what justified all the extra complexity of doing it right after.

Reply:
It wasn't inline because I wanted a way that didn't require hardcoding the names of the false positives, but I haven't thought of one yet.